releases/release-7.3.0.md

```diff
@@ -0,0 +1,352 @@
+---
+title: TiDB 7.3.0 Release Notes
+summary: Learn about the new features, compatibility changes, improvements, and bug fixes in TiDB 7.3.0.
+---
+
+# TiDB 7.3.0 Release Notes
+
+Release date: xx xx, 2023
+
+TiDB version: 7.3.0
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.3/quick-start-with-tidb) | [Installation packages](https://www.pingcap.com/download/?version=v7.3.0#version-list)
+
+7.3.0 introduces the following major features. The rest of the release (detailed in the Details section) was a series of enhancements to query stability in TiDB server and TiFlash. These are more miscellaneous in nature and not user-facing so they are not included in this section dedicated to release highlights:
```

Comment:
```suggestion
7.3.0 introduces the following major features. In addition to that, 7.3.0 also includes various improvement features (described in the [Feature details](#feature-details) section) to enhance query stability in TiDB server and TiFlash.
```

Reply:
"These are more miscellaneous in nature and not user-facing so they are not included in this section dedicated to release highlights:" is not needed and it distracts user attention.